### ISSUE TITLE
beforeFeature / afterFeature

### ISSUE THREAD

**Issue by nayaramoura** [Codeception/CodeceptJS#428]
#### What are you trying to achieve?

There are some actions that I need to execute after running all tests in a feature. 
I am wondering if we already have something like "beforeFeature / afterFeature".

#### What do you get instead?

I've already analyzed the before/after and the beforeSuite/afterSuite. The first one doesn't work for my case because it executes once for each scenario. The second doesn't have access to the browser.

So, basically, I want a command that behaves like before/after but runs only once at the and of the feature execution.

### Details

* CodeceptJS version: 0.4.8
* NodeJS Version: 6.2.1
* Operating System: IOS
* Nightmare version: 2.7.0

**Comment by DavertMik** [comment]
Yes, browser is started automatically before a test. You can start it manually:
Consider using `beforeSuite` with `manualStart: true` option enabled. 
Then you will need to create a custom method like `I.startBrowser()`: 

```js
startBrowser() {
  return this.helpers["WebDriverIO"]._startBrowser()
}
```

And this should work in `beforeSuite` context:

```js
BeforeSuite((I) => {
  I.startBrowser();
});
```
Also add `restart: false` option, to keep browser up between tests

**Comment by akrueger** [comment]
@DavertMik will this still work for `Nightmare` even though it's the `WebDriverIO` helper? It seems the OP is using Nightmare (as am I), but I don't see `manualStart` in the helper object for Nightmare.

Thanks

**Comment by akrueger** [comment]
Ah, I believe I see how it works now for Nightmare:

`CodeceptJS/lib/helper/Nightmare.js`:

```
_beforeSuite() {
    if (!this.options.restart) {
      return this._startBrowser();
    }
}
```
...
```
_startBrowser() {
    this.browser = this.Nightmare(this.options);
    this.browser.on('dom-ready', () => this._injectClientScripts());
    this.browser.on('console', (type, message) => {
      this.debug(`${type}: ${message}`);
    });

    if (this.options.windowSize) {
      let size = this.options.windowSize.split('x');
      return this.browser.viewport(parseInt(size[0]), parseInt(size[1]));
    }
}
```

So I set `restart: false` in place of `manualStart: false` and then created a custom helper function but used `Nightmare` Helper method rather than `WebDriverIO`:

```
'use strict';

class Utility extends Helper {
  startBrowser() {
    return this.helpers["Nightmare"]._startBrowser()
  }
}

module.exports = Utility;
```

That worked well for me.

**Comment by APshenkin** [comment]
Closing due inactivity
